Readme.md

```diff
@@ -2,10 +2,14 @@ A tiny utility to post current directory contents to http://plnkr.co/edit
 
 ## Anonymous Usage
 
+0. Add `bin` project folder to your path
 1. Go the the dir you want to plunk.
 2. Run `plunk`
 
-Or `plunk --dir path/to/dir --desc "A string describing the plunk" --tags "comma,separated,tags"`.
+Or `plunk --dir path/to/dir --desc "A string describing the plunk" --tags "comma,separated,tags" --glob "pattern"`.
+Specify default parameters in `defaults.json`.
+
+To run on Windows, use `plunk.bat` instead of `plunk`.
```

Comment:
It is not possible to do it as simple as that:
```
@ECHO OFF
cd /d %~dp0
node plunk $*
```
...because --dir option would not work properly in this case (it would use script's root as base path, and we need to change it to cwd, which is %~dp0).

All this hacking around in the script is to support proper passing of dir option to the underlying node script (change it, and make sure other parameters as passed as is regardless of order).

cmd scripts are "fun" to write, so I am not saying my script is the simplest way to do it, but it worked for me so I decided to share :)